zpages/tracez.go

```diff
@@ -392,3 +392,57 @@ func writeTextTraces(w io.Writer, data traceData) {
 	}
 	tw.Flush()
 }
+
+type summaryPageData struct {
+	Header             []string
+	LatencyBucketNames []string
+	Links              bool
+	TracesEndpoint     string
+	Rows               []summaryPageRow
+}
+
+type summaryPageRow struct {
+	Name    string
+	Active  int
+	Latency []int
+	Errors  int
+}
+
+func (s *summaryPageData) Len() int           { return len(s.Rows) }
```

Comment:
We use Go1.8+ and can use `sort.Slice` which is available https://github.com/golang/go/blob/cceee685be4436c8fd417e9c248429b8625b978d/api/go1.8.txt#L243